src/search/src/main.rs

```diff
@@ -76,26 +81,34 @@ fn get_emoji_dict(search: &str, path: &Path) -> Result<Dict<Vec<String>>, Error>
 			continue;
 		}
 
-		emojis.append(&mut get_emojis_in_show(search, item_path_buf.as_path())?);
+		// emojis.append(&mut get_emojis_in_show(search, item_path_buf.as_path())?);
 		
 		let folder_name = item_path_buf.as_path().file_name().unwrap().
 							to_str().unwrap();
-		let emojis_in_show = get_emojis_in_show(search, item_path_buf.as_path())?;
+		let emojis_in_show = get_emojis_in_show(search, item_path_buf.as_path(), gif)?;
 		
 		dict.add(folder_name.to_string(), emojis_in_show);
 	}
 	Ok(dict)
 }
 
-fn get_emojis_in_show(search: &str, path: &Path) -> Result<Vec<String>, Error> {
+fn get_emojis_in_show(search: &str, path: &Path, gif: &str) -> Result<Vec<String>, Error> {
```

Comment:
consider using boolean for `gif`

Reply:
Will not be doing this at this time. Perhaps if Rust had an enum, I would use that, but we need the ability to search for both as well as "yes" or "no".

Reply:
Just made an enum.